plugins/woocommerce/.wp-env.json

```diff
@@ -18,12 +18,17 @@
             "port": 8086,
             "plugins": [
                 ".",
-                "https://downloads.wordpress.org/plugin/akismet.zip"
+                "https:\/\/downloads.wordpress.org\/plugin\/akismet.zip",
```

Comment:
Could you expand a bit on why we're escaping these?

Reply:
It was just out from the [UPDATE_WP_JSON](https://github.com/woocommerce/woocommerce/pull/37804/files#diff-10b57384d8b2ab1505258a047d529b6f0fa4b14a01daaa0f87169984de5e3b87R87) class. Made the update to not escape slashes.